src/gui/render.cpp

```diff
@@ -851,7 +851,7 @@ static void init_render_settings(Section_prop& secprop)
 	        "    options instead.");
 
 #if C_OPENGL
-	string_prop = secprop.Add_path("glshader", always, "crt-auto");
+	string_prop = secprop.Add_string("glshader", always, "crt-auto");
```

Comment:
This is probably a good change since the auto CRT shaders are not paths but maybe do some quick regression testing for the case where this does point to a shader file on the disk.

Reply:
I have, relative and absolute paths work.